### ISSUE TITLE
Create a footer component 

### ISSUE THREAD

**Issue by SoulNaturalist** [SoulNaturalist/BookList#147]
![image](https://github.com/SoulNaturalist/BookList/assets/58743402/f2ca8929-b79e-42d5-b981-127ff8c16e72)

**Comment by ravikr126** [comment]
@SoulNaturalist can you assign this to me ,but I don't know your written language I can write in English if you like

**Comment by SoulNaturalist** [comment]
@ravikr126 
yes, of course, write in English, you could just make identical inscriptions just in English or just fish the text with the same length as in the original, if you want, I can translate for you what is written there to make it more convenient to place the text, etc
